Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -52,5 +53,70 @@ static bool IsCalendarReadOnly(CalendarAccess accessLevel)
         static async Task PlatformRequestCalendarReadAccess() => await Permissions.RequireAsync(PermissionType.CalendarRead);
 
         static async Task PlatformRequestCalendarWriteAccess() => await Permissions.RequireAsync(PermissionType.CalendarWrite);
+
+        public static async Task<IReadOnlyList<IEvent>> PlatformGetEventsAsync(string calendarId = null, DateTimeOffset? startDate = null, DateTimeOffset? endDate = null)
+        {
+            await Permissions.RequireAsync(PermissionType.CalendarRead);
+
+            var eventsUri = CalendarContract.Events.ContentUri;
+            var eventsProjection = new List<string>
+            {
+                CalendarContract.Events.InterfaceConsts.Id,
+                CalendarContract.Events.InterfaceConsts.CalendarId,
+                CalendarContract.Events.InterfaceConsts.Title,
+                CalendarContract.Events.InterfaceConsts.Description,
+                CalendarContract.Events.InterfaceConsts.EventLocation,
+                CalendarContract.Events.InterfaceConsts.AllDay,
+                CalendarContract.Events.InterfaceConsts.Dtstart,
+                CalendarContract.Events.InterfaceConsts.Dtend,
+                CalendarContract.Events.InterfaceConsts.Duration,
+                CalendarContract.Events.InterfaceConsts.HasAlarm,
+                CalendarContract.Events.InterfaceConsts.HasAttendeeData,
+                CalendarContract.Events.InterfaceConsts.HasExtendedProperties,
+                CalendarContract.Events.InterfaceConsts.Status,
+                CalendarContract.Events.InterfaceConsts.Rrule,
+                CalendarContract.Events.InterfaceConsts.Rdate
+            };
+            var calendarSpecificEvent = string.Empty;
+            if (!string.IsNullOrEmpty(calendarId))
+            {
+                calendarSpecificEvent = $"{CalendarContract.Events.InterfaceConsts.CalendarId}={calendarId} AND ";
```

Comment:
Generally speaking, I'm not a big fan of leaving string literals in code. I would suggest creating a const for `AND` and using it across the board.

I would probably try to create something similar for the `deleted != 1` part, as it's not super obvious what it does. I would probably name this whole part of query something like `NotDeletedEventsWhereClause` - if you can find a better wording go for it.